web/src/demo_components/progress.dart

```diff
@@ -9,13 +9,6 @@ UiFactory<ProgressProps> Progress;
 
 @Props()
 class ProgressProps extends UiProps {
-  /// The id for the [Progress] component.
-  ///
-  /// If left unspecified one will be auto-generated for you to ensure
-  /// that the [caption] element is properly linked for accessibility purposes.
-  @override
-  String get id;
```

Comment:
Could the `@virtual` annotation be used on the super field to allow this? (See [meta package](https://pub.dartlang.org/packages/meta) 1.0.4 changelog)